stylesheets/_component.notifications.scss

```diff
@@ -81,14 +108,38 @@
     display: block;
     float: left;
     padding: 10px;
+    width: 100%;
+
+    &:last-of-type {
+        border-bottom: 0;
+    }
 
     &:hover {
         background-color: color(background, light);
     }
+
+    &.is-empty {
+        color: color(text, light);
+        text-align: center;
+
+        p {
+            margin-bottom: 0;
+        }
+
+        i {
+            font-size: 3em;
+        }
+    }
+
+    // prevent large images from cropping outside the container
+    img {
+        max-width: 100% !important;
```

Comment:
No way to use a class here / could we chain to lose `!important` or are you overriding inline style?